jobs/deploy-webapp.groovy

```diff
@@ -955,7 +955,10 @@ onMaster('4h') {
 
       try {
          stage("Await first smoke test") {
-            if (SERVICES) {
+            // Only verify smoke tests if we're deploying a non-static service
+            // Static service changes have already had e2e tests run on them
+            // during the PR process, so this is unnecessary.
```

Comment:
This is an exaggeration -- these tests run against the commit + master (and possibly + the prior commit), which is exactly the kind of situation that we're worried about when we run these smoke tests in the first place: that a bad merge or something will cause problems we couldn't detect by just testing the commit itself in isolation.